source/UsersServer/docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+version: '3.8'
+services:
+  mongodb:
+    image: mongo:6-jammy
+    container_name: courses-mongo
+    restart: unless-stopped
+    ports:
+      - '27017:27017'
+    environment:
+      - MONGO_INITDB_ROOT_USERNAME=user
+      - MONGO_INITDB_ROOT_PASSWORD=pass
+      - MONGO_INITDB_DATABASE=CoursesService
```

Comment:
since this isn't actually the db name we use in the code and it still works maybe we can just drop this env var.
also if we keep it rename it to UserService